### PR TITLE
Organize folders

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
- Organizes the `components/` folder structure to a more standard practice
- Removes the `useDefaultColorMaps` property
- Exports `ColorMaps` variable, an object of `{[colorMap]: {name: "", path: ""} }` form

`ColorMaps` currently contains grayscale, natural, and rgb. Should we only expose grayscale and rgb as the defaults? Perhaps add the haline and thermal color maps?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,7 +142,6 @@ function App() {
         scale: "1 -1 1",
         rotation: "-55 0 0",
       }}
-      useDefaultColorMaps={false}
     />
   )
```
Comment (mcmcgrath13) [inline README.md:146]: the readme should probably document that the default color maps are exported and can be used if needed

File: src/components/ClipControls/ClipControls.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import styled from "styled-components";
 import { Range } from "rc-slider";
 import "rc-slider/assets/index.css";
 
-import Section from "./Section.jsx";
+import Section from "../Section/Section.jsx";
```
Comment (mcmcgrath13) [inline src/components/ClipControls/ClipControls.jsx:6]:
```suggestion
import Section from "../Section";
```
Can you do this if you add an index to the Section folder and make Section the default epxort?

File: src/components/VolumeViewer/VolumeViewer.jsx [hunk 3]
```diff
@@ -4,24 +4,22 @@ import styled from "styled-components";
...
   DEFAULT_MODEL,
   DEFAULT_TRANSFER_FUNCTION,
   DEFAULT_SLIDERS,
-} from "../constants/constants";
+} from "../../constants/constants";
 
-import Controls from "./controls/Controls.jsx";
-import AframeScene from "./AframeScene.jsx";
+import Controls from "../Controls/Controls.jsx";
+import AframeScene from "../AframeScene/AframeScene.jsx";
```
Comment (mcmcgrath13) [inline src/components/VolumeViewer/VolumeViewer.jsx:13]: same thought on these imports

File: src/constants/constants.js [hunk 4]
```diff
@@ -5,11 +5,13 @@ import rgb from "../images/rgb.png";
...
-];
+const naturalColorMap = { name: "Natural", path: natural };
+const rgbColorMap = { name: "RGB", path: rgb };
+const ColorMaps = {
+  grayscale: grayscaleColorMap,
+  natural: naturalColorMap,
+  rgb: rgbColorMap,
+};
 
 const DEFAULT_COLOR_MAP = grayscaleColorMap;
```
Comment (mcmcgrath13) [inline src/constants/constants.js:16]: the switching this to be a string will be a later PR?

File: src/index.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
-import VolumeViewer from "./components/VolumeViewer.jsx";
+import VolumeViewer from "./components/VolumeViewer";
+import { ColorMaps } from "./constants/constants";
```
Comment (mcmcgrath13) [inline src/index.js:2]: is `constants` the only thing in the `constants/` dir? if so, maybe just make it the `index` so you don't need the double slash?

Comment (RobertGemmaJr) [reply src/index.js:2]: Yes it's the only thing, good call

File: src/components/VolumeViewer/VolumeViewer.jsx (hunk 3 above)
Comment (RobertGemmaJr) [reply src/components/VolumeViewer/VolumeViewer.jsx]: Done (:

File: src/constants/constants.js (hunk 4 above)
Comment (RobertGemmaJr) [reply src/constants/constants.js:16]: Yeah I'd like to get it as a prop inside of model first, I'm working on that right now in another branch

File: README.md (hunk 1 above)
Comment (RobertGemmaJr) [reply README.md:156]: I added a note and the images to the readme

File: README.md [hunk 6]
```diff
@@ -119,18 +114,25 @@ VolumeViewer.defaultProps = {
...
   transferFunction: [{ x: 0, y: 0 }, { x: 1, y: 1 }],
-  useDefaultColorMaps: true,
   useTransferFunction: true,
 };
 
 ```
 
+## ColorMaps
+
+The `ColorMaps` export is an object containing some default color maps that may be useful for your project. There is a grayscale, "natural", and rgb color map:
```
Comment (mcmcgrath13) [inline README.md:124]: why is natural in quotes, but not the other two?

**Review by mcmcgrath13 — APPROVED** [review]
🌈

File: README.md (hunk 6 above)
Comment (RobertGemmaJr) [reply README.md:124]: Oops, will remove

**Pull request merged**